include/domains/wallinDomain.hpp

```diff
@@ -85,14 +80,14 @@ namespace ghost
     inline pair<int, int> getStartingTile()	const { return startingTile; }
     inline pair<int, int> getTargetTile()	const { return targetTile; }
            
-    inline int		 getNberRows()	const { return nRow_; }
-    inline int		 getNberCols()	const { return mCol_; }
+	inline int		 getNberRows()	const { return maxY_; }
+	inline int		 getNberCols()	const { return maxY_; }
     inline bool		 hasFailure()	const { return !failures_.empty(); }
     inline mapFail	 failures()	const { return failures_; }
 
-    inline pair<int, int> lin2mat( int p )	      const {return make_pair(p / mCol_, p % mCol_);}
-    inline int		  mat2lin( int row, int col ) const {return row * mCol_ + col;}
-    inline int		  mat2lin( pair<int, int> p ) const {return p.first * mCol_ + p.second;}
+	inline pair<int, int> lin2mat(int p)	      const { return make_pair(p % maxY_, p / maxY_); }
```

Comment:
Do you mean the space after "(" and before ")"? I have never seen this spacing policy :P
In fact, Visual Studio removes automatically those spaces after a copy & paste...